about.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/about.css">
+  <script src="https://kit.fontawesome.com/1ada02b4f0.js" crossorigin="anonymous"></script>
+  <title>About the tourism </title>
+</head>
+
+<body>
+  <header>
+    <div class="top-navbar">
+      <ul>
+        <li>
+          <a href="#"><i class="fa fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa fa-facebook"></i></a>
+        </li>
+        <li><a href="#">Celebrities</a></li>
+        <li><a href="#">Vlog</a></li>
+        <li><a href="#"> Logout</a></li>
+      </ul>
+    </div>
+    <div class="main-navbar flex-center">
+      <div class="logo">
+        <a href="index.html"><img src="./assets/business-logo.jpg" alt="fashion logo" /></a>
+      </div>
+      <nav class="navbar">
+        <div class="hamburger">
+          <img src="./assets/hamburger.svg" alt="Menu Hamburger" />
+        </div>
+        <ul class="nav-links">
+          <li class="close-menu-btn">
+            <button type="submit">
+              <span>&times;</span>
+            </button>
+          </li>
+          <li><a href="index.html" id="current">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Models</a></li>
+          <li><a href="#">Tickets</a></li>
+          <li id="leehar-line-lg" class="flex-center d-none d-block-lg"><a href="#">CC Campaign</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <section class="about-section">
+    <div class="about-wrapper">
+      <div class="about-section-header">
+        <h1 class="about-info">
+          CREATIVE COMMONS GLOBAL SUMMIT 2015
+        </h1>
+      </div>
+      <div class="about-text">
+        <p>
+          The Global Submit brings together the community of experts, academics and activities who comprise of creative commons affiliate network in a different country every two years. This year, we hope to expand our invitation list includes organisations and individuals.
+        </p>
+      </div>
+
+      <div class="about-contact">
+        <p>Kindly leave your comments, reviews and questions on our Email </p>
+        <a href="#" class="about-adress">
+          ccsummit2015@cckorea.org
```

Comment:
- [ ]  GREAT JOB on storing and using the speakers in the  `js` files.  However, your style needs some touchup on both the first sections of the home page and the about page.  because one of the requirements says all page elements have similar spacing, borders, shadows, and paddings to the original design.



Kindly I would recommend following the original design, check the difference down here👇  👇 
## The template
![image](https://user-images.githubusercontent.com/65068771/147105191-51d95640-9221-466b-9854-b9f715e22dad.png)

Your first home section | your first about section
------------------------| -----------------------
![image](https://user-images.githubusercontent.com/65068771/147106152-d50560c8-fbd6-421d-b057-4bd556e44afa.png) | ![image](https://user-images.githubusercontent.com/65068771/147105866-913cc39e-e77b-42c2-b74a-ddc9a4c55e0a.png)

Reply:
@andersonlebon. Thanks so much
It does not matter to be the same but due to your comment I've used the same `background and font-size`.